client/i18n/locales/arabic/intro.json

```
@@ -1340,15 +1340,21 @@
       },
       "learn-how-to-ask-and-share-about-educational-and-professional-background": {
         "title": "Learn How to Ask and Share About Educational and Professional Background",
-        "intro": ["", ""]
+        "intro": [
+          "In this course, you'll learn how to ask about another person's educational and professional backgrouns and you'll learn how to share information about your background as well."
```

Just pointing out a typo to open an issue just now.

```suggestion
          "In this course, you'll learn how to ask about another person's educational and professional backgrounds and you'll learn how to share information about your background as well."
```